test/browser/support/F2F_YOTI_START_00_SCHEMA.json

```diff
@@ -0,0 +1,178 @@
+                                    "type": "array",
+                                    "items": [
+                                        {
+                                            "type": "object",
+                                            "properties": {
+                                                "latitude": {
+                                                    "type": "number"
+                                                },
+                                                "longitude": {
+                                                    "type": "number"
+                                                }
+                                            },
+                                            "required": [
+                                                "latitude",
+                                                "longitude"
+                                            ]
+                                        }
+                                    ]
+                                }
+                            },
+                            "required": [
+                                "address",
+                                "post_code",
+                                "location"
+                            ]
+                        }
+                    ]
+                }
+            },
+            "required": [
+                "evidence",
+                "post_office_details"
+            ]
+        },
+        "restricted": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "array",
+                    "items": [
+                        {
+                            "type": "object",
+                            "properties": {
+                                "nameParts": {
+                                    "type": "array",
+                                    "items": [
+                                        {
+                                            "type": "object",
+                                            "properties": {
+                                                "type": {
+                                                    "type": "string"
+                                                },
+                                                "value": {
+                                                    "type": "string"
+                                                }
+                                            },
+                                            "required": [
+                                                "type",
+                                                "value"
+                                            ]
+                                        }
+                                    ]
+                                }
+                            },
+                            "required": [
+                                "nameParts"
+                            ]
```

Comment:
As per KIWI-1781 changes, you may have to update the YOTI_START schemas to include device_information field?